CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 ### 0.10.0 (Major Release)
 
+#### Subrecord variables in the forms
```

Comment:
There are some typos/missing words in this changelog section...
I would probably move most of the detail here to the upgrade guide - I think that's where I want to know about the details of changes?